beacon-chain/verification/result.go

```diff
@@ -3,6 +3,35 @@ package verification
 // Requirement represents a validation check that needs to pass in order for a Verified form a consensus type to be issued.
 type Requirement int
 
+func (r Requirement) String() string {
```

Comment:
Probably overkill here, but add a test that ensures all `Requirements` in `GossipSidecarRequirements` have a String() other than unknown

Reply:
done